.gitignore

```diff
@@ -28,4 +28,5 @@
 **yarn-error.log*
 **yarn.lock*
 
-.idea/
\ No newline at end of file
+.idea/
+.env
```

Comment:
@phawazzzy I think we are leaving .env not hidden intentionally, so this should not be in the file

Reply:
Yes